docs/installation.md

```diff
@@ -254,12 +254,17 @@ This is **the ONLY way we recommend**. We do not recommend install `file` via Sc
 
 Most users already have Git installed, and Yazi is also hosted via Git, so this usually isn't an issue. But if you really don't have/want to install it, the [`mime-ext.yazi`](https://github.com/yazi-rs/plugins/tree/main/mime-ext.yazi) plugin can help, which use a extension database instead of relying on the `file(1)` binary.
 
+Installing Yazi on Windows Terminal does not automatically install nerd-fonts, you need to manually change the font.Open Windows Terminal `Settings -> Profiles -> Defaults -> Additional settings -> Appearance -> Font face`. Changed from the default `Cascadia Mono` to `CaskaydiaMono Nerd Font Mono`.
```

Comment:
We can't assume that users are using Windows Terminal. Please remove it or move it to a separate page, where you can list more terminal configuration options and provide a link to that page.